be/src/olap/wrapper_field.cpp

```diff
@@ -53,14 +53,17 @@ WrapperField* WrapperField::create(const TabletColumn& column, uint32_t len) {
 }
 
 WrapperField* WrapperField::create_by_type(const FieldType& type) {
-    Field* rep = Field::create_by_type(type);
-    if (rep == nullptr) {
+    bool is_string_type = (type == OLAP_FIELD_TYPE_CHAR
```

Comment:
We have change this function's logic in 6d040a33afa292e2c2e75239d77f82b77fcc3cfa
1. you should add TYPE_BITMAP_UNION as is_string_type
2. you should change `be/src/olap/stream_index_common.cpp` `ColumnStatistics::init`'s logic
3. this can create wrapper field for string type

Reply:
The type of bitmap_union is varchar.